packages/rum-core/src/common/constants.js

```diff
@@ -159,6 +159,11 @@ const TRUNCATED_TYPE = '.truncated'
  */
 const KEYWORD_LIMIT = 1024
 
+/**
+ * Session timeout is set to 30 minutes.
+ */
+const SESSION_TIMEOUT = 30 * 60000
```

Comment:
```suggestion
const SESSION_TIMEOUT = 30 * 60
```

Reply:
This value is in milliseconds, I'll change the comment to make it clear.